Mathlib/Logic/Nonempty.lean

```diff
@@ -28,6 +28,10 @@ theorem Nonempty.forall {α} {p : Nonempty α → Prop} : (∀ h : Nonempty α,
 theorem Nonempty.exists {α} {p : Nonempty α → Prop} : (∃ h : Nonempty α, p h) ↔ ∃ a, p ⟨a⟩ :=
   Iff.intro (fun ⟨⟨a⟩, h⟩ ↦ ⟨a, h⟩) fun ⟨a, h⟩ ↦ ⟨⟨a⟩, h⟩
 
+@[simp low]
```

Comment:
Please add a comment explaining why this is here; you gave one reason, another is to not override `exists_prop`